modules/net-vpc/routes.tf

```diff
@@ -88,3 +88,43 @@ resource "google_compute_route" "vpn_tunnel" {
   tags                = each.value.tags
   next_hop_vpn_tunnel = each.value.next_hop
 }
+
+resource "google_compute_route" "private" {
```

Comment:
an we leverage the existing route resource, by defining a local with the private ones, and merging with the routes variable?